setup.py

```diff
@@ -5,7 +5,7 @@
 INSTALL_REQUIRES = ["alembic", "click>=7.0"]
 EXTRAS_REQUIRE = {
     "tests": ["pytest"],
-    "lint": ["fourmat==0.2.0", "pre-commit~=1.18"],
+    "lint": ["fourmat~=1.20.0", "pre-commit~=1.18"],
```

Comment:
Hm, this doesn't look right. `pre-commit` should be upgraded `~=1.20.0` and fourmat should be upgraded to `==0.4.0` 🤔